package.json

```diff
@@ -18,54 +18,57 @@
     "schemas/"
   ],
   "dependencies": {
-    "@hapi/boom": "^9.1.0",
-    "@hapi/hapi": "^19.1.1",
-    "@hapi/vision": "^6.0.0",
-    "@microfleet/core": "^17.3.0",
-    "@microfleet/transport-amqp": "^15.0.0",
-    "@microfleet/validation": "^9.0.0",
+    "@babel/parser": "7.22.0",
+    "@hapi/boom": "^10.0.1",
+    "@hapi/hapi": "^21.3.2",
+    "@hapi/vision": "^7.0.1",
+    "@microfleet/core": "^19.2.6",
+    "@microfleet/transport-amqp": "^17.4.0",
+    "@microfleet/validation": "^11.1.3",
```

Comment:
microfleet requires many more plugins now:

`@microfleet/core-types`
`@microfleet/plugin-logger`
`@microfleet/plugin-validator`
`@microfleet/plugin-router`
`@microfleet/plugin-router-amqp`
`@microfleet/plugin-amqp`
`@microfleet/plugin-prometheus`

make sure to adjust core set of plugins to include new ones (namely prometheus, router-amqp)